src/main/java/frc/team2412/robot/subsystems/ArmSubsystem.java

```diff
@@ -131,28 +119,24 @@ public static class ArmConstants {
 		 * Scoring Wrist Angle
 		 */
 		public static enum PositionType {
-			UNKNOWN_POSITION(0.212, 0.122, 0.122, 0.477, 0.477),
```

Comment:
We should probably update the comment above to only have the retracted wrist angle and not the retracted wrist angle (cone edition).